packages/erlang-ls/package.yaml

```
@@ -13,7 +13,7 @@ categories:
 
 source:
   # renovate:datasource=github-tags
-  id: pkg:github/erlang-ls/erlang_ls@0.52.0
+  id: pkg:github/erlang-ls/erlang_ls@1.0.0
```

```suggestion
  id: pkg:github/erlang-ls/erlang_ls@1.1.0
```

Version 1.1.0 was released today: https://github.com/erlang-ls/erlang_ls/releases